docs/spark-configuration.md

```diff
@@ -63,15 +63,20 @@ Iceberg supplies two implementations:
+| Property                                                      | Values                        | Description                                                                                                                                                                                                                              |
+|---------------------------------------------------------------|-------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| spark.sql.catalog._catalog-name_.type                         | `hive`, `hadoop` or `rest`    | The underlying Iceberg catalog implementation, `HiveCatalog`, `HadoopCatalog`, `RESTCatalog` or left unset if using a custom catalog                                                                                                     |
+| spark.sql.catalog._catalog-name_.catalog-impl                 |                               | The underlying Iceberg catalog implementation.                                                                                                                                                                                           |
+| spark.sql.catalog._catalog-name_.io-impl                      |                               | The custom FileIO implementation.                                                                                                                                                                                                        |
+| spark.sql.catalog._catalog-name_.metrics-reporter-impl        |                               | The custom MetricsReporter implementation.                                                                                                                                                                                               |
+| spark.sql.catalog._catalog-name_.default-namespace            | default                       | The default current namespace for the catalog                                                                                                                                                                                            |
+| spark.sql.catalog._catalog-name_.uri                          | thrift://host:port            | Hive Metastore URI (default from `hive-site.xml`) for hive catalog and REST URI for rest catalog                                                                                                                                         |
```

Comment:
should also add example for a REST URI

Reply:
Sure I can add one such example for rest catalog like hive and Hadoop catalog above